### ISSUE TITLE
Description repeated when layers are added/removed

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/greenhouse-effect#386]
**Test device**
MacBook Air M1 chip

**Operating System**
14.1.2

**Browser**
Safari 17.1.2

**Problem description**
For https://github.com/phetsims/qa/issues/1033, On the Layer Model Screen when using VO + keyboard nav OR a11y view + keyboard nav:
When the number of absorbing layers is changed, description is repeated --ex. Layer 1 added above surface. Layer 1 added above surface

**Steps to reproduce**
1. In a11y View on the Layer Model Screen (or in the sim with VO on), Tab to Absorbing Layers vertical number spinner and press the Right Arrow key once 
2. Press the Right Arrow Key again
3. Press the Left Arrow Key

**Visuals**
<img width="705" alt="Screenshot 2024-01-31 at 2 06 36 PM" src="https://github.com/phetsims/greenhouse-effect/assets/87318828/7f758a1d-5e12-447c-bc8c-1d909236c4d9">
<img width="705" alt="Screenshot 2024-01-31 at 2 07 26 PM" src="https://github.com/phetsims/greenhouse-effect/assets/87318828/792d524a-f0ac-41bd-a98c-2ad0e92ab278">

<img width="705" alt="Screenshot 2024-01-31 at 2 19 37 PM" src="https://github.com/phetsims/greenhouse-effect/assets/87318828/524d2ddb-39d0-40d2-b165-bad4ed6f697f">

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Greenhouse Effect‬
URL: https://phet-dev.colorado.edu/html/greenhouse-effect/1.3.0-dev.3/phet/greenhouse-effect_all_phet.html
Version: 1.3.0-dev.3 2024-01-23 21:58:15 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.2.1 Safari/605.1.15
Language: en-US
Window: 1356x710
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by Nancy-Salpepi** [comment]
The same thing happens when the % of Infrared Absorbance is changed. The "All layers now absorbing X%....." statement gets repeated.

<img width="669" alt="Screenshot 2024-01-31 at 2 27 04 PM" src="https://github.com/phetsims/greenhouse-effect/assets/87318828/a99638d1-cc8a-47a5-b526-9c25e2eab622">

**Comment by arouinfar** [comment]
Good catch @Nancy-Salpepi! I was able to reproduce in the a11y view and VO.

@jbphet the double alert occurs for keyboard interaction, but not for mouse interaction.

**Comment by jbphet** [comment]
I've done some investigation, and I can see that the alert is actually being performed twice in the keyboard interaction case, once by `LayerModelModelAlerter` and once by the `a11yCreateContextResponseAlert` option for the `NumberPicker` in `AbsorbingLayersControl`.  The latter is not triggered via mouse interaction but _is_ triggered via keyboard interaction.  The former occurs regardless of the type of interaction.

Removing the  `a11yCreateContextResponseAlert` option to the `NumberPicker` seems like the obvious solution, but I'd like to run this by @jessegreenberg before doing that, since he has much more experience with description than I.

**Comment by jessegreenberg** [comment]
Yes, I think that is right! I was curious why didn't notice this before and found the commit message for https://github.com/phetsims/greenhouse-effect/commit/68982a7569e8f10a09b65e0c9f88b71a7581539e 
> move context responses from a11yCreateContextResponseAlert options to the stepped alerter for better sequencing, see https://github.com/phetsims/greenhouse-effect/issues/374

The `a11yCreateContextResponseAlert`s were removed from the `SolarIntensityControl` and the `SurfaceAlbedoControl` and I think we just forgot to remove from the `AbsorbingLayersControl`.

**Comment by jbphet** [comment]
Cool.  I'll go ahead and remove the one from the number picker (`a11yCreateContextResponseAlert `).

**Comment by KatieWoe** [comment]
This looks ok in the a11y view of the sim.

**Comment by Nancy-Salpepi** [comment]
Also fixed with VO.

**Comment by Nancy-Salpepi** [comment]
Going to reopen because I see the same issue with changes to IR absorbance:
<img width="676" alt="Screenshot 2024-02-16 at 12 49 46 PM" src="https://github.com/phetsims/greenhouse-effect/assets/87318828/40ef1590-5207-4e08-b109-e59d731817df">

**Comment by jbphet** [comment]
Good catch @Nancy-Salpepi.  The repeated alerts about "All layers now absorbing..." had the same root cause as the "Layer added" ones, namely that there was a `a11yCreateContextResponseAlert` option passed in to a UI component (a slider in this instance), and there was also code to create this alert based on periodic updates.  I've removed the `a11yCreateContextResponseAlert` option and the code that went with it.  I've verified that there is now only a single alert generated for both keyboard and mouse interaction.

I also searched the greenhouse-effect code for any other usages of `a11yCreateContextResponseAlert` and didn't find any, so there shouldn't be any other double-alerts that are caused by this.
